apps/api/src/routers/tiers.py

```diff
@@ -22,14 +22,18 @@
 logger = logging.getLogger("root")
 
 
-@router.get("/{id)")
-def get_tiers(q: TierGetRequest = Depends()) -> list[Tier]:
-    return db.get_tier(q.profile_id)
+@router.get("/{id}")
+def get_tiers(id: UUID) -> list[Tier]:
+    response = db.get_tiers(id)
+    if not response:
+        raise HTTPException(404, "tiers information not found")
```

Comment:
make this endpoint call `get_tier(id)` function instead, and rename the endpoint function to `get_tier` (line 26)